lib/fs.js

```diff
@@ -1585,6 +1585,10 @@ function encodeRealpathResult(result, options, err) {
   }
 }
 
+// This is removed from the fs exports in lib/module.js in order to make
+// sure that this stays internal.
+const realpathCacheKey = fs.realpathCacheKey = Symbol('realpathCacheKey');
```

Comment:
I would say that as soon as possible we should move this into a `internal/fs` once we're able to actually make that change.